### PR TITLE
fix for macos finder issue

### PR DESCRIPTION
**Pull request by Kethku**
@shaunsingh @fredizzimo in that case I vote we merge this.

## What kind of change does this PR introduce?
- Fix

## Did this PR introduce a breaking change? 
_A breaking change includes anything that breaks backwards compatibility either at compile or run time._
- Yes, please list breaking changes

### CODE REVIEW

**Comment by fredizzimo** [comment]
Like I mentioned in https://github.com/neovide/neovide/issues/447, I think `platform_which` and `platform_exists` needs to be implemented. It probably works now due to this hack
https://github.com/neovide/neovide/blob/8542333c940b0549c82da1ff89f6b9bd11f73c53/src/main.rs#L201-L220

But if those are implemented, then that should no longer be needed, and it appears as if that hack does not work on all systems anyway.

File: src/bridge/mod.rs [hunk 1]
```diff
@@ -109,6 +109,16 @@ fn build_nvim_cmd_with_args(bin: &str) -> Command {
         cmd.args(&["$SHELL", "-lc", &argstring]);
         return cmd;
     }
+
+    #[cfg(macos)]
```
Comment (fredizzimo) [inline src/bridge/mod.rs:113]: Is this clippy error, something we need to be concerned of?

Comment (Kethku) [reply src/bridge/mod.rs:113]: I'll look into it

**Comment by Kethku** [comment]
@shaunsingh any chance you could try the currently pushed version? I don't have a mac so I can't test...

**Comment by shaunsingh** [comment]
I apologize for the delay, I was having a few issues with my GitHub account. Works great, changes lgtm!

**Pull request merged**